crates/pallet-domains/src/staking.rs

```diff
@@ -181,8 +181,6 @@ pub struct Operator<Balance, Share, DomainBlockNumber> {
     pub nomination_tax: Percent,
     /// Total active stake of combined nominators under this operator.
     pub current_total_stake: Balance,
-    /// Total rewards this operator received this current epoch.
-    pub current_epoch_rewards: Balance,
```

Comment:
This was kept there to track the total rewards by the operator in the given epoch. We do emit events for this but having it seperately means, its just one runtime call down the line if we ever need it.

I would still prefer to keep it as is

Reply:
All the rewards are tracked in `DomainStakingSummary::current_epoch_rewards`, and distributed to the operator during epoch transition with tax deducted.

Before this PR, the reward is moved to `Operator:: current_epoch_rewards`and then `Operator::current_total_stake` during epoch transition, meaning `Operator:: current_epoch_rewards` is always zero besides during epoch transition, and we still need to worry all the time to add `current_epoch_rewards` to `current_total_stake` before doing any computation 🤷‍♂️